src/lib/operators/import.cpp

```diff
@@ -91,6 +116,8 @@ std::shared_ptr<const Table> Import::_on_execute() {
     ChunkEncoder::encode_all_chunks(table, chunk_encoding_spec);
   }
 
+  Hyrise::get().storage_manager.add_table(_tablename, table);
```

Comment:
Can you add a comment why the changed order has an impact?

Reply:
Since I requested the change, let me make a draft:
`// We create statistics when tables are added to the storage manager. As statistics can be expensive to create and their creation benefits from dictionary encoding, we add the tables after they are encoded.`